CHANGELOG.md

```diff
@@ -38,6 +38,10 @@
  * Note that `email_verified == not email_verification_required`.
  * A data migration will be necessary.
 
+## v8.1.1
+
+* Bugfix: Don't show "passwords do not match" when the first password is invalid.
```

Comment:
This should mention that it is a cherry-pick of a later release, otherwise it looks very odd that this is in the history twice.

That said, I don't see this elsewhere in this file... are you sure that this was fixed before?